AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test.py

```diff
@@ -29,6 +29,21 @@ def detect_product(sql_connection, platform, target):
         hits = hits + 1
     assert hits == 1
 
+def compile_test_case_name(request):
+    """
+    Compile a test case name for consumption by the TIAF python coverage listener gem.
+    @param request: The fixture request.
+    """
+    try:
+        test_case_prefix = "::".join(str.split(request.node.nodeid, "::")[:2])
+        test_case_name = "::".join([test_case_prefix, request.node.originalname])
+        callspec = request.node.callspec.id
+        compiled_test_case_name = f"{test_case_name}[{callspec}]"
+    except Exception as e:
+        logging.warning(f"Error reading test case name for TIAF. {e}")
+        compiled_test_case_name = "ERROR"
+    return compiled_test_case_name
```

Comment:
Hmm is this a carbon copy of the function call from `hydra_test_utils.py`? We probably don't want to add this repeated code and it would be better to move this to a utility file instead if possible

Reply:
That's a good idea. I've looked into it and I think I can put it in ly_test_tools editor_utils. Going to put it there and run some tests.

Reply:
That sounds great thanks! 😄

Reply:
Tests have passed, this change has worked.

Reply:
Excellent! 🙂